src/main/kotlin/racingcar/RacingCarController.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar
+
+import racingcar.domain.RacingCarGame
+import racingcar.domain.SeedMakerImpl
+import racingcar.view.InputView
+import racingcar.view.ResultView
+
+class RacingCarController(private val inputView: InputView, private val resultView: ResultView) {
+
+    private val seedMaker = SeedMakerImpl()
```

Comment:
```suggestion
class RacingCarController(
    private val inputView: InputView,
    private val resultView: ResultView,
    private val seedMaker: SeedMaker = SeedMakerImpl()
) {
```

`default parameter`를 사용해서 `SeedMakerImpl`을 넣어주는 방법도 있을 것 같아요!